contents/spotlight/startup-puzzle.md

```diff
@@ -0,0 +1,79 @@
+
+“On [our pricing page](https://puzzle.io/pricing) we make this pretty clear,” says Scott. “We have a plan for formation-stage founders, a plan for startups who have raised a little funding, and a scale plan for companies which are generating revenue.
+
+Each plan also reflects the pain points a user would deal with at that stage too and addresses it with a product feature. Monthly compliance checklists in the free plan, for example, help formation-stage founders get started quickly — but give way in the scale-up plan to AI automation and new processes.
+
+> **Putting it into practice:** If you aren’t using delineated pricing plans then it can still be helpful to think of your broader [pricing principles](/handbook/engineering/feature-pricing). Ask yourself: how do these principles work for your different user segments? 
+
+## 4. Reinforce your ICP with internal teams
+
+As you grow it’s important not to drift from your ICP or to let teams’ understanding of it become corrupted. You need to constantly remind everyone in your company who you are building for and how your product meets their needs — otherwise you risk diluting your offering. 
+
+“When I joined Puzzle I was very happy to hear people talking about our ICP,” says Scott. “Then I saw there was some confusion and some of what was said was incorrect. Some people thought our ICP is a company, while others thought it was a person, and there were some different ideas floating around.”
+
+At PostHog, we’ve solved this issue with [a public handbook](/handbook) which documents [our ICP](/handbook/who-we-are-building-for) — but Puzzle took a different approach.  
+
+“We have these all-hands meetings where everybody has to put their hand up and swear that our ICP is a founder and not an accountant,” says Scott. 
+
+> **Putting it into practice:** There are lots of ways to disseminate information internally, from intranets to incorporating it into employee onboarding. Whichever route you take though, the key is consistency. 
+
+## 5. Once you start, don’t stop
+
+Once you’ve established your ICP you must constantly maintain it. You need to always be speaking to users, working to understand their concerns, and analyzing the data behind their usage. In doing so you can find new segments to target, new needs to address, and new product improvements to make.
+
+“We started looking at our biggest user drop-offs and found many ICP users were dropping off when they had to connect their bank,” says Scott. “We reached out to these users directly and talked to them. We had to try and understand their concerns.”
+
+Scott’s team learned that customers were becoming frustrated by delays during this connection step. As a result they improved the content on the page, added new loading animations, and surfacing data more quickly — an ICP-informed change which had an immediate impact on overall activation. 
```

Comment:
Scott’s team learned that customers were becoming frustrated by delays before they could see their dashboard after completing onboarding. As a result they brought users into the app as it was loading so they could quickly see what they were waiting for. This reduced time to value dramatically and was a clear example of an ICP-informed change which had an immediate impact on overall activation.